src/Generated/EnumRegister.inc

```diff
@@ -1,4 +1,4 @@
-static const ZydisShortString STR_REGISTER[] =
+static const ZydisShortString STR_REGISTERS[] =
```

Comment:
This name clashes with another `STR_REGISTER` somewhere in the generated formatter strings when doing amalgamated builds, so I had to rename it. I'll push a commit to also change that in `gen_registers.py` once this PR is merged.